components/bsx/Offer/CollectionOffers.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
```

Comment:
I bet this loading modal dialog does not work

Reply:
remove it